source/localizable/cps_orders_api/index.html.md.erb

```diff
@@ -0,0 +1,128 @@
+---
+title: cps_orders_api
+---
+
+<%= partial 'partials/page_locales' %>
+
+# Orders API <%= edit_link %>
+
+Orders API allows Skroutz Merchants to perform actions on their CPS orders,
+such as accepting or rejecting an order.
+
+<%= partial 'partials/toc' %>
+
+## Setup
+
+### Obtaining an API token
+
+In order to be able to use the API, you need to generate an **API token** from within CPS orders settings
+page in merchant's panel (Merchants > Παραγγελίες καταστήματος > CPS orders API).
+
+> <i class="fa fa-exclamation-triangle"></i>
+> There can only be one active **API key** per shop at a certain time.
+> Creating a new API key will expire any previous one.
+
+### Authorization
+
+Pass in a valid **API token** in the `Authorization` header, prefixed with `Bearer `
+as in the following curl example.
+
+A specific `Accept` header is also required. For more information on API requests check the [API overview](/api/v3/).
+
+<pre class="terminal">
+curl -X GET https://<%= settings.api_domain %>/merchants/cps/orders/1234567 \
+  -H 'Accept: application/vnd.<%= settings.site_name %>+json; version=3.0' \
+  -H 'Authorization: Bearer your_access_token_here'
+</pre>
+
+## Endpoints
+
+
+### Fetch a single order
+
+To fetch a specific order, you have to provide the order `code` with which you have originally identified 
+your order in your shop, as a URL parameter.
+
+When an order is in a state that can be rejected, `reject_options` attribute is included in the
+response, providing all the available options for issuing "reject" requests for the order.
+
+<pre class="terminal">
+  GET /merchants/cps/orders/:code
+</pre>
+
+<%= render_recording :merchants_cps_orders_show %>
+
+
+### Reject a single order
+
```

Comment:
I think it would be helpful to mention here that there are two ways to reject orders:

> In order to reject an order, you **must** include a reject reason. The reject reason can be either one of the predefined ones we support or it can be a custom one that the user wants to provide.

Then, the following two sections can elaborate on the two possible options.